docs/serving/autoscaling/autoscale-go/README.md

```diff
@@ -220,44 +219,44 @@ customization (32 minutes).
 
 1. Send 60 seconds of traffic maintaining 100 concurrent requests.
 
-   ```bash
-   hey -z 60s -c 100 \
-     "http://autoscale-go.default.1.2.3.4.sslip.io?sleep=100&prime=10000&bloat=5"
-   ```
+     ```bash
+     hey -z 60s -c 100 \
+       "http://autoscale-go.default.1.2.3.4.sslip.io?sleep=100&prime=10000&bloat=5"
+     ```
 
 1. Send 60 seconds of traffic maintaining 100 qps with short requests (10 ms).
 
-   ```bash
-   hey -z 60s -q 100 \
-     "http://autoscale-go.default.1.2.3.4.sslip.io?sleep=10"
-   ```
+     ```bash
+     hey -z 60s -q 100 \
+       "http://autoscale-go.default.1.2.3.4.sslip.io?sleep=10"
+     ```
 
 1. Send 60 seconds of traffic maintaining 100 qps with long requests (1 sec).
 
-   ```bash
-   hey -z 60s -q 100 \
-     "http://autoscale-go.default.1.2.3.4.sslip.io?sleep=1000"
-   ```
+     ```bash
+     hey -z 60s -q 100 \
+       "http://autoscale-go.default.1.2.3.4.sslip.io?sleep=1000"
+     ```
 
 1. Send 60 seconds of traffic with heavy CPU usage (~1 cpu/sec/request, total
    100 cpus).
 
-   ```bash
-   hey -z 60s -q 100 \
-     "http://autoscale-go.default.1.2.3.4.sslip.io?prime=40000000"
-   ```
+     ```bash
+     hey -z 60s -q 100 \
+       "http://autoscale-go.default.1.2.3.4.sslip.io?prime=40000000"
+     ```
 
 1. Send 60 seconds of traffic with heavy memory usage (1 gb/request, total 5
    gb).
 
-   ```bash
-   hey -z 60s -c 5 \
-     "http://autoscale-go.default.1.2.3.4.sslip.io?bloat=1000"
-   ```
+     ```bash
+     hey -z 60s -c 5 \
+       "http://autoscale-go.default.1.2.3.4.sslip.io?bloat=1000"
+     ```
 
 ## Cleanup
 
-```
+```bash
 kubectl delete --filename docs/serving/autoscaling/autoscale-go/service.yaml
```

Comment:
another one where you could use `-f` > `--filename` but it's up to you if you wanna fix that in this PR